frontend/src/utils/institute/formRules.js

```diff
@@ -189,6 +189,10 @@ const minLength = ( length, message = 'Must be at least ${length} characters') =
   return v => !v || v.length === length || message;
 };
 
+const penIsValid = (message = 'PEN must be a 9 digit number') => {
```

Comment:
Are we including check digit validation?

Reply:
if you are talking about checking for a 9 digit number, that's what the regex in the next line is doing. Are you referring to the same validCheckDigit in the PenUtil in the api? If so, I would need to add this.

Reply:
Yes that's right the `validCheckDigit`. Ideally we layer it on to ensure they're typing in an actual PEN and not just any 9 digit number.